### PR TITLE
Rules window

### PR DESCRIPTION
**Pull request by nyan-ko**
Create the basic rule window display class. No user interaction logic specific to the rules window, so no controllers needed.

### CODE REVIEW

File: src/main/java/ui/windows/RulesDisplay.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        add(buttonPanel, BorderLayout.PAGE_END);
+    }
+
+    public static void main(String[] args) {
+        RulesDisplay display = new RulesDisplay();
+        JFrame frame = new JFrame("RulesDisplay");
+        frame.setSize(400, 400);
+        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        frame.add(display);
+        frame.setVisible(true);
```
Comment (meiiph) [inline src/main/java/ui/windows/RulesDisplay.java:48]: Is there a way to choose font size? Maybe we should make the font bigger for better accessibility.

**Review by meiiph — APPROVED** [review]
Looks good to keep moving, just see if there's a way to make the font size bigger later. :)

File: src/main/java/ui/windows/RulesDisplay.java (hunk 1 above)
Comment (nyan-ko) [reply src/main/java/ui/windows/RulesDisplay.java:48]: Good feature to implement if we have time later on. The JLabel.setFont method seems to solve this, seen [here](https://stackoverflow.com/questions/2715118/how-to-change-the-size-of-the-font-of-a-jlabel-to-take-the-maximum-size)

**Review by SMateen21 — APPROVED** [review]
Looks good, I'd just add some light Javadocs to each method, but nothing major.

**Review by lilyxmeng — APPROVED** [review]
Looks good

**Pull request merged**